lib/balena-lint.ts

```diff
@@ -265,14 +265,19 @@ export const lint = async (passedParams: any) => {
 			'Ignore linting config files in project directory and its parents',
 		)
 		.describe('e', 'Override extensions to check, eg "-e js -e jsx"')
-		.boolean('typescript', 'Lint typescript files instead of coffeescript')
-		.boolean('fix', 'Attempt to automatically fix lint errors')
-		.boolean('no-prettier', 'Disables the prettier code format checks')
-		.boolean(
+		.describe('typescript', 'Lint typescript files instead of coffeescript')
```

Comment:
I'd prefer to use the [.option](https://github.com/yargs/yargs/blob/master/docs/api.md#optionkey-opt) style of yargs in order to keep all the bits relevant to a flag in one place, ie
```suggestion
		.option('typescript', {
			describe: 'Lint typescript files instead of coffeescript'
			type: 'boolean'
		})
```